### ISSUE TITLE
Background update failed covers widget

### ISSUE THREAD

**Issue by GlytchMeister** [breezy-weather/breezy-weather#1421]
### Steps to reproduce

Have widget and background services notifications enabled

Have widget as a regular notification

Have background services as a silent notification

### Expected behavior

Background services update failed notification remains in silent notifications section, widget remains visible without dismissing silent background services notification

### Actual behavior

Background services update failed notification joins with widget and takes priority over the widget, hiding it until Background services notification is dismissed

### Weather source used

Multiple

### Breezy Weather version

Stable 5.2.8

### Android version

Android 15, GrapheneOS 2024110700

### Device

Pixel 8

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
I don't think it's a bug as I never documented anywhere this behavior, nor intended a specific behavior.

In either case, I can't reproduce, this is what it looks like on Android 15 on my side (enable widget, then have a background failure, or the other way around):
![IMG_20241112_132841](https://github.com/user-attachments/assets/144b29df-6717-49b5-a32b-6956d580b7f5)

**Comment by GlytchMeister** [comment]
> I don't think it's a bug as I never documented anywhere this behavior, nor intended a specific behavior.
> 
> In either case, I can't reproduce, this is what it looks like on Android 15 on my side

No, that's exactly what I'm talking about.  The widget is "behind" (for lack of a better word) the background failure notification.

I'd hoped that background failure notification was supposed to stay in the silent section of the notification pulldown while the widget stays visible in the main section.  Whenever this notification appears, I have to tap the "2 v" button to expand and separate them, then swipe away just the background failure one.  I can't just swipe, because that temporarily dismisses the widget, too, so then I have to unlock my phone and go open the app to check the temperature and hourly forecast instead of just glancing at my phone without unlocking it... Which defeats the purpose of the widget.

If this background failure notification stayed in the silent section, I wouldn't have to go through this whole song and dance.  Or, if background failures could be made a separate section within the notification settings for background services, I'd be able to turn just those off without having to turn off background services off entirely.

**Comment by papjul** [comment]
If your goal is just to not display the background error notification on your lock screen, go to App info > Notifications > Background Services > Lock screen => Don't display

**Comment by GlytchMeister** [comment]
> If your goal is just to not display the background error notification on your lock screen, go to App info > Notifications > Background Services > Lock screen => Don't display

Well, that's great for the lock screen,  but I'd like the widget to work properly in the pulldown, too.  Not just the lock screen.

Notifications can have subcategories in the settings, can't they?  Couldn't failures be split off from the regular background services, within the background services notifications section of the notifications settings?

I just want the widget to stop getting covered by other breezy notifications.
